src/app/Topology/styles/base.css

```diff
@@ -50,10 +50,14 @@ Below CSS rules only apply to Topology components
   aspect-ratio: 1;
   border-radius: 50%;
   border: 1px solid var(--pf-global--palette--black-300);
-  background-color: #fff;
+  background-color: var(--pf-global--palette--white);
   margin: 1em;
 }
 
```

Comment:
Lets try this:

```diff
diff --git a/src/app/Topology/styles/base.css b/src/app/Topology/styles/base.css
index 3f77a668..c17b356f 100644
--- a/src/app/Topology/styles/base.css
+++ b/src/app/Topology/styles/base.css
@@ -55,7 +55,7 @@ Below CSS rules only apply to Topology components
 }
 
 :where(.pf-theme-dark) .sample-node-donut__node-wrapper {
-  background-color: var(--pf-global--BackgroundColor--300);
+  background-color: var(--pf-global--BackgroundColor--dark-300);
 }
 
 .sample-node-donut__node-wrapper .sample-node-donut__node-icon {
@@ -96,7 +96,7 @@ Below CSS rules only apply to Topology components
 }
 
 :where(.pf-theme-dark) .sample-node-donut__node-wrapper .sample-node-donut__status-indicator {
-  background-color: var(--pf-global--BackgroundColor--300);
+  background-color: var(--pf-global--BackgroundColor--dark-300);
 }
 
 .sample-node-donut__node-wrapper .sample-node-donut__status-indicator svg {
@@ -113,6 +113,11 @@ Below CSS rules only apply to Topology components
   box-shadow: 0 0.25rem 0.5rem 0rem rgba(3, 3, 3, 0.16), 0 0 0.25rem 0 rgba(3, 3, 3, 0.06);
   font-weight: 700;
   word-break: break-all;
+  background-color: var(--pf-global--palette--white);
+}
+
+:where(.pf-theme-dark) .sample-node-donut__node-label {
+  background-color: var(--pf-global--BackgroundColor--dark-300);
 }
```

![Screenshot from 2023-03-29 02-43-31](https://user-images.githubusercontent.com/68053619/228448217-0022856e-9aac-4f56-a334-f0842c648f52.png)